tox.ini

```diff
@@ -6,4 +6,6 @@ passenv = *
 deps=
   -rrequirements.txt
 commands=
-  py.test --cov plaid -s -rxs ./tests/
+  coverage run --source=plaid/,tests/ -m pytest --strict {posargs}
+  coverage report -m --show-missing --fail-under 96
+  flake8 .
```

Comment:
This sets a hard lower limit on test coverage and fails on pep8 issues.
 I think it's good practice, but please confirm this is what we want.